### PR TITLE
Ctrl scroll zoom

### LINKED ISSUE #430: Zoom with Ctrl+Scroll
Like browsers and Code

### PR DESCRIPTION
**Pull request by jeremypw**
Fixes #430

### CODE REVIEW

File: src/Widgets/TerminalWidget.vala [hunk 1]
```diff
@@ -171,6 +173,42 @@ namespace Terminal {
...
+                            increment_size ();
+                            return Gdk.EVENT_STOP;
+
+                        case Gdk.ScrollDirection.DOWN:
+                            decrement_size ();
+                            return Gdk.EVENT_STOP;
+
+                        case Gdk.ScrollDirection.SMOOTH:
+                            double delta_x, delta_y;
+                            event.get_scroll_deltas (out delta_x, out delta_y);
```
Comment (danrabbit) [inline src/Widgets/TerminalWidget.vala]: If you're only interesting in `delta_y` I think you can just do 
```suggestion
                            var delta_y = event.delta_y;
```

**Comment by danrabbit** [comment]
Wow summing delta is way more elegant than the solution I came up with elsewhere. Definitely gonna steal that!

One thing is that it doesn't look like this updates the zoom level in the menu :)

**Comment by jeremypw** [comment]
Oops - forgot about updating the menu - thanks.  This was more or less cut and pasted from Files code.

File: src/Widgets/TerminalWidget.vala (hunk 1 above)
Comment (jeremypw) [reply src/Widgets/TerminalWidget.vala]: Thanks - done

**Pull request merged**